File: 002-Recursion/009.checkPalindrome.py
def isPalindrome(s,start,end):
    if start >= end:
        return True
    if s[start] != s[end]:
        return False
    return isPalindrome(s,start+1,end-1)

def main():
    s = 'axbca'
    ans = isPalindrome(s,0,len(s) - 1)
    print(ans)

if __name__ == '__main__':
    main()

File: Geeks-For-Geeks/002-Recursion/009.checkPalindrome.js
function isPalindrome(s, start, end) {
  if (start >= end) {
    return true;
  }
  if (s[start] != s[end]) {
    return false;
  }
  return isPalindrome(s, start + 1, end - 1);
}

const s = 'harshit';
const isP = isPalindrome(s, 0, s.length - 1);
console.log(isP);
